docs/concepts/what-is-an-activity-task.md

```diff
@@ -2,9 +2,28 @@
 id: what-is-an-activity-task
 title: What is an Activity Task?
 sidebar_label: Activity Task
-description: An Activity Task is a Task that contains the context needed to make progress with an Activity Execution.
+description: An Activity Task contains the context needed to make an Activity Task Execution.
 tags:
   - explanation
 ---
 
-An Activity Task is a Task that contains the context needed to make progress with an Activity Execution.
+An Activity Task contains the context needed to proceed with an [Activity Task Execution](/docs/concepts/what-is-an-activity-task-execution).
+
+Activity Tasks are meant to run an [Activity Definition](/docs/concepts/what-is-an-activity-definition) once.
+If an Activity Task Execution fails, the Activity Task is retried in accordance with its Retry Policy.
+
+#### Scheduling Activity Tasks
+
+Activity Tasks are assigned to an Activity Task Queue upon an [ActivityTaskScheduled Event](/docs/concepts/what-is-an-activity-task#scheduling-activity-tasks).
```

Comment:
I think its more of a correlation -
An Activity Task comes into existence when it is scheduled - and the Event is added to the Workflow's Event History.